### PR TITLE
Fill DB and getInitState  / Add message to DB

### PR DESCRIPTION
**Pull request by hotspurs**

### CODE REVIEW

File: app/components/MessageComposer/index.jsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default class MessageComposer extends React.Component {
     const text = this.state.text.trim();
     if (text) {
       this.props.newMessage({
-        id: 1,
+        // id: 1,
```
Comment (dimaip) [inline app/components/MessageComposer/index.jsx]: Do not commit commented out code, especially without comments. In this case just remove it, will add it later if needed.

**Comment by dimaip** [comment]
Please update the readme, stating that we now require mongodb to be installed. Readme should always tell the exact steps needed to get things running at any time.

**Comment by Nitive** [comment]
@dimaip maybe write it to wiki?

**Comment by dimaip** [comment]
@Nitive I think all essential steps needed to get the project running should be in readme, and extra stuff, such as specs, can go to wiki

**Comment by Nitive** [comment]
@dimaip Okey, no problem

**Comment by Nitive** [comment]
++

**Comment by SeqviriouM** [comment]
+

**Comment by Nitive** [comment]
It closes #64

**Pull request merged**